site/sfguides/src/data_collaboration_native_app/data_collaboration_native_app.md

```diff
@@ -0,0 +1,630 @@
+CREATE OR REPLACE TABLE cc_default_unscored_data LIKE cc_default_training_data;
+ALTER TABLE cc_default_unscored_data DROP COLUMN "target";
+
+ALTER WAREHOUSE query_wh SET warehouse_size=MEDIUM;
+
+USE WAREHOUSE query_wh;
+
+COPY INTO cc_default_unscored_data 
+  FROM @quickstart_cc_default_unscored_data 
+  FILE_FORMAT = (FORMAT_NAME= 'parquet_format') 
+  MATCH_BY_COLUMN_NAME=CASE_INSENSITIVE;
+
+ALTER WAREHOUSE query_wh SET warehouse_size=XSMALL;
+```
+
+This app requires the consumer to explicitly provide a reference to a table in the Snowflake instance it is installed as an input. References are a way for Native App consumers to be explicit in their permissioning to any existing object in a Snowflake account that is required. This ensures that Native Applications run as "isolated objects" within a Consumer Account, and must be granted explicit permission to access other objects. More information can be foind [here](https://docs.snowflake.com/en/developer-guide/native-apps/requesting-refs). We can achieve referencing through the UI, or running the following SQL.
+
+```SQL
+-- Bind the reference to a table in our own Snowflake Account. This can be done via SQL or the UI
+CALL credit_card_prediction_app.app_code.register_single_reference('RAW_TABLE' , 'ADD', SYSTEM$REFERENCE('TABLE', 'NATIVE_APP_DEMO.NATIVE_APP_DEMO.CC_DEFAULT_UNSCORED_DATA', 'PERSISTENT', 'SELECT'));
+
+-- Check that the reference worked
+SHOW REFERENCES IN APPLICATION credit_card_prediction_app;
+```
+
+Next let's run through the commands and see if they are working as expected. Note that we are running code inside an isolated Application object. This application object will eventually be shared with our consumer (Snowbank).
+
+```SQL
+-- This should be empty, as we have not called the procedure to populate the table yet
+SELECT * FROM CREDIT_CARD_PREDICTION_APP.APP_CODE.RAW_TABLE_VIEW;
+
+-- Now we call the procedure
+CALL CREDIT_CARD_PREDICTION_APP.APP_CODE.CREATE_TABLE_FROM_REFERENCE_RAW();
+
+-- And the table should now be populated
+SELECT * FROM CREDIT_CARD_PREDICTION_APP.APP_CODE.RAW_TABLE_VIEW LIMIT 10;
+SELECT COUNT(*) FROM CREDIT_CARD_PREDICTION_APP.APP_CODE.RAW_TABLE_VIEW;
+
+-- Next we call the procedure to do the feature engineering. This will take 5 mins to complete
+CALL CREDIT_CARD_PREDICTION_APP.APP_CODE.CC_PROFILE_PROCESSING();
+
+-- The feature table should now be populated
+SELECT * FROM CREDIT_CARD_PREDICTION_APP.APP_CODE.TRANSFORMED_TABLE_VIEW LIMIT 10;
+
+-- Next we call the procedure that infers our model
+USE WAREHOUSE TRAINING_WH;
+CALL CREDIT_CARD_PREDICTION_APP.APP_CODE.CC_BATCH_PROCESSING();
+
+-- And we can now see the scored table
+SELECT * FROM CREDIT_CARD_PREDICTION_APP.APP_CODE.SCORED_TABLE_VIEW LIMIT 10;
+```
+We are satisfied with the testing. The next step is to distribute the app.
+
+<!-- ------------------------ -->
+## Provider Account (Zamboni) - Distribute Native App
```

Comment:
@sfc-gh-tbuchhorn Rename this to "Distribute Native App" and add "Provider Account (Zamboni)" sub-header underneath.